### ISSUE TITLE
Download data set problem

### ISSUE THREAD

**Issue by zcswdt** [AliaksandrSiarohin/video-preprocessing#15]
Hi
Thank you for awesome repo.
When I use the `python load_videos.py --metadata vox-metadata.csv --format .mp4 --out_folder vox --workers 8` command to download the vox data set, an error ''Can not load video na8-QEFmj44, broken link'' appears. And there is no video in the newly created folder.

**Comment by AliaksandrSiarohin** [comment]
Single video is missing or all?

**Comment by zcswdt** [comment]
> Single video is missing or all?

than you reply,all videos cannot be downloaded,No background process。
![image](https://user-images.githubusercontent.com/43515926/100173120-d7184a00-2f04-11eb-8922-700d12ecee80.png)

**Comment by AliaksandrSiarohin** [comment]
Can you access them on YouTube? Can you load some videos using YouTube-dl?

**Comment by zcswdt** [comment]
> Can you access them on YouTube? Can you load some videos using YouTube-dl?

I can access YouTube by using local windows and over the wall, but I am using the code running on the Linux server, I don’t know if I can access YouTube. How to use download videos via YouTube-dl? 
Is this through the command line `YouTube-dl https://www.youtube.com/watch?v=VXP7-x_w2eg`?

**Comment by AliaksandrSiarohin** [comment]
Yes, ```youtube-dl https://www.youtube.com/watch?v=VXP7-x_w2eg```. If you launch from a folder then ```./youtube-dl https://www.youtube.com/watch?v=VXP7-x_w2eg```.

**Comment by zcswdt** [comment]
> Yes, `youtube-dl https://www.youtube.com/watch?v=VXP7-x_w2eg`. If you launch from a folder then `./youtube-dl https://www.youtube.com/watch?v=VXP7-x_w2eg`.

Thank you for your help, i can download the video using youtube-dl by opening vpn. But I encountered a new problem, the error is as follows:
![image](https://user-images.githubusercontent.com/43515926/100302073-7f491400-2fd4-11eb-9168-9a4d01781e01.png)

**Comment by damengdameng** [comment]
> > Yes, `youtube-dl https://www.youtube.com/watch?v=VXP7-x_w2eg`. If you launch from a folder then `./youtube-dl https://www.youtube.com/watch?v=VXP7-x_w2eg`.
> 
> Thank you for your help, i can download the video using youtube-dl by opening vpn. But I encountered a new problem, the error is as follows:
> ![image](https://user-images.githubusercontent.com/43515926/100302073-7f491400-2fd4-11eb-9168-9a4d01781e01.png)

你好 请问你的问题解决了嘛？

**Comment by zcswdt** [comment]
> > > Yes, `youtube-dl https://www.youtube.com/watch?v=VXP7-x_w2eg`. If you launch from a folder then `./youtube-dl https://www.youtube.com/watch?v=VXP7-x_w2eg`.
> > 
> > 
> > Thank you for your help, i can download the video using youtube-dl by opening vpn. But I encountered a new problem, the error is as follows:
> > ![image](https://user-images.githubusercontent.com/43515926/100302073-7f491400-2fd4-11eb-9168-9a4d01781e01.png)
> 
> 你好 请问你的问题解决了嘛？

yes，no problem

**Comment by damengdameng** [comment]
@zcswdt 请问方便加个qq嘛？ 有几个关于数据集的问题想请教下 我的是：937173518 谢谢！

**Comment by zcswdt** [comment]
> @zcswdt 请问方便加个qq嘛？ 有几个关于数据集的问题想请教下 我的是：937173518 谢谢！

加了。

**Comment by dengsutao** [comment]
> > @zcswdt 请问方便加个qq嘛？ 有几个关于数据集的问题想请教下 我的是：937173518 谢谢！
> 
> 加了。

hello, i met the same problem as yours, can i contact you by QQ? my QQ is 651922058, thanks for your help!

**Comment by zcswdt** [comment]
> > > @zcswdt 请问方便加个qq嘛？ 有几个关于数据集的问题想请教下 我的是：937173518 谢谢！
> > 
> > 
> > 加了。
> 
> hello, i met the same problem as yours, can i contact you by QQ? my QQ is 651922058, thanks for your help!

my qq is 810190882

**Comment by dengsutao** [comment]
i have sent you some messages by qq～ On 01/28/2021 11:33, zcswdt wrote: 







@zcswdt 请问方便加个qq嘛？ 有几个关于数据集的问题想请教下 我的是：937173518 谢谢！



加了。



hello, i met the same problem as yours, can i contact you by QQ? my QQ is 651922058, thanks for your help!



my qq is 810190882



—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

[

{

"@context": "http://schema.org",

"@type": "EmailMessage",

"potentialAction": {

"@type": "ViewAction",

"target": "https://github.com/AliaksandrSiarohin/video-preprocessing/issues/15#issuecomment-768774876",

"url": "https://github.com/AliaksandrSiarohin/video-preprocessing/issues/15#issuecomment-768774876",

"name": "View Issue"

},

"description": "View this Issue on GitHub",

"publisher": {

"@type": "Organization",

"name": "GitHub",

"url": "https://github.com"

}

}

]

**Comment by Amazingren** [comment]
> 

哈喽老哥，我想问问你那里现在有这个vox-celeb的数据集吗。我下载过程中也一直出现这个问题，折腾了好久都没有解决～

**Comment by william-ljz** [comment]
> > Single video is missing or all?
> 
> than you reply,all videos cannot be downloaded,No background process。 ![image](https://user-images.githubusercontent.com/43515926/100173120-d7184a00-2f04-11eb-8922-700d12ecee80.png)

哥们你知道voxceleb2数据集咋下吗

**Comment by Amazingren** [comment]
您的来信已经收到，稍后进行回复，谢谢~

**Comment by wmxlovextt** [comment]
> > 单个视频丢失还是全部丢失？
> 
> 比你回复，所有视频都无法下载，没有后台进程。 ![image](https://user-images.githubusercontent.com/43515926/100173120-d7184a00-2f04-11eb-8922-700d12ecee80.png)

I have the same problem. can not load video ******* breaken link, none of the videos can be downloaded

**Comment by wmxlovextt** [comment]
> Do you know how to download the voxceleb dataset

**Comment by wmxlovextt** [comment]
> > 
> 
> 哈喽老哥，我想问问你那里现在有这个vox-celeb的数据集吗。我下载过程中也一直出现这个问题，折腾了好久都没有解决～

Hello hello, I also have a problem downloading the voxceleb dataset, always can not load video ****broken link

**Comment by Qia98** [comment]
有人下载过完整的voxceleb2数据集吗？可以分享一下吗（不方便公开的话可以email我，16170217981@163.com）？目前我只能找到voxceleb1的数据集


Has anyone downloaded the complete VoxCeleb2 dataset? Could you please share it with me? If it's not convenient to share publicly, you could email me at 16170217981@163.com. Currently, I can only find the VoxCeleb1 dataset.
